src/data/pequiv.lean

```diff
@@ -303,6 +339,7 @@ end pequiv
 namespace equiv
 variables {α : Type*} {β : Type*} {γ : Type*}
 
+/-- Turns an `equiv` into a `pequiv` in the canonical way. -/
```

Comment:
I found "the canonical way" not super clear at first; maybe something like "... a `pequiv` that maps all elements/the entire type"?

Reply:
Is this better? Canonical is a very overused word, you're right :b